esp-hal/src/soc/esp32/mod.rs

```diff
@@ -27,6 +27,7 @@ macro_rules! chip {
 }
 
 /// A link to the Technical Reference Manual (TRM) for the chip.
+#[doc(hidden)]
```

Comment:
I kind of wanted the same visibility as `chip!` for these, they are equally (not) interesting for the users I think :)